<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hbx</class>
 <widget class="QMainWindow" name="hbx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hbx</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#hbx {
background-color: black;
}

#frameDisplay {
	border-radius: 6px;
	background-color: #5b0202;
}

#displayL1 {
	color:#c00000;
}

#displayL2 {
	color:#c00000;
}

#txtInstruction {
	border-radius: 6px;
	background-color: #5b0202;
	color:#c00000;
}

#centralWidget {
border: 3px solid #3e005b;
padding: 10px;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #000000, stop: 1 #270039);
}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #161616, stop: 1 #323030);
border-width: 1px;
border-color: #330000;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 24px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 150px;
min-height: 24px;
max-height: 32px;
}

QPushButton:pressed {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #323030, stop: 1 #161616);
border-width: 1px;
border-color: #330000;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 24px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 150px;
min-height: 24px;
max-height: 32px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="displayL2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier 10 Pitch</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="displayL1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier 10 Pitch</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txtInstruction">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>391</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMode">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGoto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>GOTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbW">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuest">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDown">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCentering">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGuiding">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>630</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Guid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSlew">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>630</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Slew</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMax">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>630</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFw">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbBw">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSetTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
